.github/workflows/main.yml

```diff
@@ -0,0 +1,52 @@
+name: Node.js CI
+
+on: [push]
+
+jobs:
+  test-node:
```

Comment:
Should we add needs lint to test-node and test-karma jobs?

Reply:
@JSAssassin ah, yes, let's add em!